### DISCUSSION TITLE
Best way to turn on and off camera?

### DISCUSSION THREAD

**Discussion by resonanttoe** [gtxaspec/wz_mini_hacks#386] [Q&A]
Hi folks,

I'm looking at a way to automate (using some HA triggers that are fine) turning on and off the camera (Wyze V3 with wz_mini). Poking around it seems that there isn't a method through cmd to do it and it seems to be controlled through icamera?

Is the best option here just to write a small script to kill/spawn  /system/bin/iCamera ?
Or is there a better method for starting/stopping the video feed?

cheers

**Comment by kohrar** [comment]
I'm not aware of any way to stop the camera from functioning like that short of buying a smart plug and powering the camera through that.  Are you trying to stop the camera from recording to the SD card / Cloud / RTSP streaming, or are you trying to accomplish something else?

You cannot kill iCamera as you suggested since the camera's hardware watchdog will reset the device and restart the firmware/iCamera again.

**Reply by resonanttoe** [reply]
Fair enough. TY for info.

My setup is RTSP fed through various plugins and that's all working fine. I can disengage the RTSP stream using the downstream mechanism (frigate in this case or dropping it at the HA level, doesn't matter, both are doable) but I thought it might be possible within the camera itself. It doesn't record to device, so its just a matter of dropping the stream.

So it may just be my lack of understanding on how they work in the Out-of-box mode. But there is a location based on/off that it has in the Wyze app. It also has a bit of an audible click when you use the Wyze app to on/off the video. Now this may be just that it stops "broadcasting" the RTSP-equivilent stream to WYZE cloud and it still is "on" in the capture video sense but I'm not really sure.
